PRs automatically failed by Travis Cl

**Describe the bug**
As of today, all PRs are automatically failed by Travis CL.

The reported error:

```
FAILURE: Build failed with an exception.
* What went wrong:
Could not determine the dependencies of task ':distTar'.
> Could not resolve all task dependencies for configuration ':runtimeClasspath'.
   > Could not resolve net.rptools.clientserver:clientserver:1.4.0.+.
     Required by:
         project :
      > Failed to list versions for net.rptools.clientserver:clientserver.
         > Could not list versions using M2 pattern 'https://jitpack.io/[organisation]/[module]/[revision]/[artifact]-[revision](-[classifier])(.[ext])'.
            > Could not get resource 'https://jitpack.io/net/rptools/clientserver/clientserver/'.
               > Could not GET 'https://jitpack.io/net/rptools/clientserver/clientserver/'. Received status code 522 from server: Origin Connection Time-out
   > Could not resolve com.github.RPTools:parser:1.5.4.
     Required by:
         project :
      > Skipped due to earlier error
   > Could not resolve com.github.RPTools:dicelib:1.5.3.
     Required by:
         project :
      > Skipped due to earlier error
```

--- a/build.gradle
+++ b/build.gradle
@@ -164,7 +164,11 @@ dependencies {
     implementation group: 'org.apache.commons', name: 'commons-collections4', version: '4.1' // https://mvnrepository.com/artifact/org.apache.commons/commons-collections4
 
     implementation 'net.java.abeille:abeille-formsrt:2.0'
-    implementation 'net.rptools.clientserver:clientserver:1.4.0.+'
+
+    // sourceControl gitRepository currently not producing classpath for Eclipse
+    // using jitpack.io for now
+    // implementation 'net.rptools.clientserver:clientserver:1.4.0.+'
+    implementation 'com.github.RPTools:clientserver:1.4.0.0'
     implementation 'org.hibernate:antlr:2.7.5H3'
 
     implementation 'commons-beanutils:commons-beanutils-core:1.8.3'

